src/support/steps/general.ts

```diff
@@ -58,6 +58,14 @@ Given('plugin is activated', async function (this: ICustomWorld) {
     await this.page.locator('a:has-text("Activate Plugin")').click();
 });
 
+/**
+ * Executes the step to activate the WP plugin.
+ */
+Given('activate {string} plugin', async function (plugin) {
+    await this.utils.gotoPlugin();
+    await this.utils.togglePluginActivation(plugin, true);
```

Comment:
2nd param will evaluate to true by default, no need to declare.
```suggestion
    await this.utils.togglePluginActivation(plugin);
```